apps/prairielearn/src/lib/workspace.ts

```diff
@@ -643,34 +643,26 @@ async function getGradedFilesFromFileSystem(workspace_id: string): Promise<strin
   }
 
   // Zip files from filesystem to zip file
-  await async.eachLimit(gradedFiles || [], config.workspaceJobsParallelLimit, async (file) => {
-    try {
-      const remotePath = path.join(remoteDir, file.path);
-      debug(`Zipping graded file ${remotePath} into ${zipPath}`);
-      archive.file(remotePath, { name: file.path });
-    } catch (err) {
-      debug(`Graded file ${file.path} does not exist`);
-    }
```

Comment:
I could see no reason that this should be `async.eachLimit`. Removing that allowed me to remove `workspaceJobsParallelLimit` from our config too!

Reply:
Agreed.